scripts/install-cni.sh

```diff
@@ -209,14 +211,14 @@ function fillSubnetsInCniSpecV2Template {
         ROUTES_REPLACEMENT+=('{"dst": "0.0.0.0/0"}')
       fi
     elif is_ipv6_range "${subnet}" ; then
-      echo "IPv6 subnet detected in .spec.podCIDRs: '${subnet:-}'"
+      log "IPv6 subnet detected in .spec.podCIDRs: '${subnet:-}'"
       POPULATE_IP6TABLES="true"
-      echo "ip6tables will be populated because IPv6 podCIDR is configured (from .spec.podCIDRs)"
+      log "ip6tables will be populated because IPv6 podCIDR is configured (from .spec.podCIDRs)"
       ipv6_subnet_configured="true"
       SUBNETS_REPLACEMENT+=("$(jq -nc --arg subnet "${subnet}" '[{"subnet": $subnet}]')")
       ROUTES_REPLACEMENT+=('{"dst": "::/0"}')
     else
-      echo "[ERROR] Subnet detected in .spec.podCIDRs '${subnet}' is not a valid IP range"
+      log "[ERROR] Subnet detected in .spec.podCIDRs '${subnet}' is not a valid IP range"
```

Comment:
Maybe it's better to introduce log level on the log handler method instead. I.e. separate methods for log_debug, log_info and log_error. Also log_error might have built-in script termination with error code.

Reply:
The current logs are pretty random... I just replaced all echo with log for the purpose of appending timestamps. Maybe more improvements later if we have capacity.

Reply:
Sounds good! It's definitely an improvement even in current state.